src/aat/resources/features/F-051/S-109.td.json

```diff
@@ -79,7 +79,7 @@
 									"pre_states": [
 										"*"
 									],
-									"post_state": null,
+									"post_states": [],
```

Comment:
the endpoint /caseworkers/{uid}/profile is used by the UI. Will this change break the UI?

Reply:
I have tested this.. nothing is broken.. I will keep an eye on this